src/options.rs

```diff
@@ -32,109 +30,112 @@ use crate::repl::OutputFormat;
 use crate::tty_password;
 use crate::watch::options::WatchCommand;
 
-pub mod describe;
-
-const MAX_TERM_WIDTH: usize = 90;
+const MAX_TERM_WIDTH: usize = 100;
 const MIN_TERM_WIDTH: usize = 50;
 
 const CONN_OPTIONS_GROUP: &str =
-    "CONNECTION OPTIONS (`edgedb --help-connect` to see full list)";
-const CLOUD_OPTIONS_GROUP: &str = "CLOUD OPTIONS";
+    "Connection Options (edgedb --help-connect to see full list)";
```

Comment:
Now is `edgedb -- --help-connect` to see the full list

Reply:
Why the double dash? `edgedb --help-connect` works just fine as before.  Am I missing something?

Reply:
Sorry, that was when running the executable through cargo which I usually do. Please ignore